nebula-logger/main/logger-engine/classes/LogEntryEventBuilder.cls

```diff
@@ -65,6 +65,7 @@ global with sharing class LogEntryEventBuilder {
             LoggingLevelOrdinal__c = loggingLevel.ORDINAL(),
             OriginType__c = ORIGIN_TYPE_APEX,
             Timestamp__c = System.now(),
+            TimestampInMillisecond__c = DateTime.now().getTime(),
```

Comment:
Hey @arvindnarsimhan - one last thought on the field name. Since this is now using the `getTime()` method instead of `milliseconds()`, I think a field name like `EpochTimestamp__c` would be more accurate/descriptive than `TimestampInMillisecond__c`. Let me know what you think.

Reply:
@jongpie Let me rename it to EpochTimestamp__c